main.go

```diff
@@ -2,8 +2,63 @@ package main
 
 import (
 	"fmt"
+	"log"
+
+	"github.com/sensu-community/sensu-plugin-sdk/sensu"
+	"github.com/sensu/sensu-go/types"
+)
+
+// Plugin ...
+type Plugin struct {
+	sensu.PluginConfig
+	Example string
+}
+
+// Options ...
+type Options struct {
+	Example sensu.PluginConfigOption
+}
+
+var (
+	plugin = Plugin{
+		PluginConfig: sensu.PluginConfig{
+			Name:     "{{ .GithubProject }}",
+			Short:    "{{ .Description }}",
+			Timeout:  10,
```

Comment:
I think we should remove `Timeout`: as @palourde pointed out it's not used in the SDK and as @echlebek pointed out, checks themselves shouldn't dictate their timeout (that's what the Sensu CheckConfig is for).